doc/tech/usecases.md

```diff
@@ -2,22 +2,44 @@
 # CNTT Use Cases
 
 ## Table of Contents
- * [5.1 Core](#5.1)
- * [5.2 Edge](#5.2)
+ * [5.1 Edge](#5.1)
+   * [5.1.1 Executive Summary](#5.1.1)
+   * [5.1.2 Objective](#5.1.2)
+   * [5.1.3 Approach & Scope](#5.1.3)
+   * [5.1.4 Principle](#5.1.4)
+   * [5.1.5 Use cases](#5.1.5)
+   * [5.1.6 Terminologies](#5.1.6)
+   * [5.1.7 SDO interlocking](#5.1.7)
+
    
 <a name="5.1"></a>
-## 5.1 Core
+## 5.1 Edge
 
+<a name="5.1.1"></a>
+### 5.1.1 Executive Summary
 
-<a name="5.1"></a>
-## 5.1 Edge
+Edge computing considered one of disruptive  technology should be considered by any Digital Service provider (DSP) as it is tightly linked with a lot of use cases that can monetize the 5G adoption , therefor a lot of CSP , Public  cloud , content provider and application developer put focus on Edge 
+
+Edge is consider one of main enabler to 5G use cases adoption  , as there are a lot of 5G challenges that Edge will support to solve them , So to achieve 5G requirements  that define by ITU-2020 for 5G , To have enhanced mobile broadband (eMBB) , massive machine-type communications (mMTCs) and ultra-reliable low-latency communications (uRLLCs) Edge is required 
```

Comment:
It appears that this is supposed to be composed of multiple sentences (see upper case letters). Need reference for ITU-2020 for 5G.  Maybe can be rewritten as in:

 Edge is considered as one of the main enablers to 5G use cases adoption.  There are a number of 5G challenges that Edge will support solve so as to implement the 5G requirements defined by "ITU-2020 for 5G."  Edge is required to support enhanced mobile broadband (eMBB), massive machine-type communications (mMTCs) and ultra-reliable low-latency communications (uRLLCs).

Reply:
done , thx